src/pages/lib/graphqlapi/query-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,582 @@ export const meta = {
+import { GetTodoQuery } from './API';
+
+const todos = await API.graphql<GraphQLQuery<GetTodoQuery>>({
+  query: queries.listTodos,
+  authMode: GRAPHQL_AUTH_MODE.AWS_IAM
+});
+```
+
+</Block>
+
+<Block name="JavaScript">
+
+```js
+import { API } from 'aws-amplify';
+import { GRAPHQL_AUTH_MODE } from '@aws-amplify/api';
+import * as queries from './graphql/queries';
+
+
+const todos = await API.graphql({
+  query: queries.listTodos,
+  authMode: GRAPHQL_AUTH_MODE.AWS_IAM
+});
+```
+
+</Block>
+</BlockSwitcher>
+
+</Accordion>
+
+## Filter list queries
+  
+As your data grows, you will need to paginate your list queries. Fortunately, this is already built in to the Amplify-generated GraphQL API.
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { API } from 'aws-amplify';
+import { listTodos } from './graphql/queries'
+import { ListTodosQueryVariables, ListTodosQuery } from './API';
+
+const variables: ListTodosQueryVariables = {
+  filter: {
+    priority: {
+      eq: 1
+    }
+  }
+};
+
+await API.graphql<GraphQLQuery<ListTodosQuery>>({ 
+  query: listTodos, variables: variables 
+});
+```
+
+</Block>
+
+<Block name="JavaScript">
+
+```js
+import { API } from 'aws-amplify';
+import { listTodos } from './graphql/queries'
+
+const variables = {
+  filter: {
+    priority: {
+      eq: 1
+    }
+  }
+};
+
+await API.graphql({ 
+  query: listTodos, variables: variables 
+});;
+```
+
+</Block>
+</BlockSwitcher>
+
+<Accordion title='How do filters work?' headingLevel='4' eyebrow='Learn more'>
+
+You can find the input schemas in the Docs pane of the GraphQL explorer or inside your autogenerated `/graphql` folder. They look like this:
+
+```graphql
+listTodos(
+  filter: ModelTodoFilterInput
+  limit: Int
+  nextToken: String): ModelTodoConnection
+
+input ModelTodoFilterInput {
+	id: ModelIDInput
+	priority: ModelIntInput
+	# ... all your other Todo fields here
+	and: [ModelTodoFilterInput]
+	or: [ModelTodoFilterInput]
+	not: ModelTodoFilterInput
+}
+```
+
+The input types in your schema indicate what kinds of filtering you can perform on them. For example, an integer field like `ModelIntInput` has this schema:
+
+```graphql
+input ModelIntInput {
+  ne: Int # "not equal to"
+  eq: Int # "equal to"
+  le: Int # "less than or equal to"
+  lt: Int # "less than"
+  ge: Int # "greater than or equal to"
+  gt: Int # "greater than"
+  between: [Int]
+  attributeExists: Boolean
+  attributeType: ModelAttributeTypes
+}
+```
+
+These filters vary based on the type of the field, but are linked to corresponding [Amazon DynamoDB queries](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Condition.html).
+
+</Accordion>
+
+### Compound filters
+
+You can combine filters with `and`, `or`, and `not` Boolean logic. Observe, in the autogenerated schema above, that `ModelTodoFilterInput` is recursive in respect to those fields. So if, for example, you wanted to filter for `priority` values of 1 OR 2, you would do this:
```

Comment:
Just noticed the "ModelTodoFilterInput' reference here is about text that is in the accordion. Suggest we change it to direct them to that accordion vs saying "above".